README.md

```diff
@@ -1,5 +1,7 @@
 # catwalk
 
+![logo](./catwalk.svg)
```

Comment:
Instead of 'logo' perhaps 'catwalk' and indexing of the page would still read catwalk